pkg/virt-launcher/virtwrap/converter/converter_test.go

```diff
@@ -803,6 +803,40 @@ var _ = Describe("Converter", func() {
 			//TODO add s390x entry with custom check of model used (disks/interfaces/controllers/devices will use different models)
 		)
 
+		Context("with ephemeral disk", func() {
+			DescribeTable("a scsi controller", func(enabled bool, arch string) {
```

Comment:
nit - Passing a `GomegaMatcher` here instead of the `bool` would be cleaner but not required.

```go
DescribeTable("a scsi controller", func(match types.GomegaMatcher, arch string) {
[..]
  Expect(domain.Spec.Devices.Controllers).To(match)
[..]
},
  Entry("should appear if enabled for amd64",
    ContainElement(
      api.Controller{
        Type: "scsi",
        Index: "0",
        Model: "virtio-non-transitional",
      },
    ),
    amd64,
  ),
)
```